src/pages/pages.11tydata.js

```diff
@@ -0,0 +1,20 @@
+import { execSync } from "node:child_process";
+
+export default {
+  permalink: "/{{ page.fileSlug }}/",
+  layout: "PageLayout.11ty.tsx",
+  eleventyComputed: {
+    lastUpdated: (data) => {
+      const result = execSync(
+        `git log -1 --pretty="format:%cI" ${data.page?.inputPath}`,
```

Comment:
this seems to break in production env. Random posts online suggest it's because cloudflare does a shallow clone so all files get the date of the last commit.

options look to be
- use github API, but need auth token?
- maybe just use git to clone the whole repo into a tmp dir and read from there?
-some sort of commit hook to store that info as content in each tracked MD file?